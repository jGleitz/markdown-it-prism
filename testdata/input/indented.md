# Test

This is an indented code block:

	public class Foo() {
		public Foo(bar) {
			System.out.println(bar);
		}
	}
