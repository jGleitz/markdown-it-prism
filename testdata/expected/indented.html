<h1>Test</h1>

<p>This is an indented code block:</p>
<pre><code>
	public class Foo() {
		public Foo(bar) {
			System.out.println(bar);
		}
	}
</code></pre>
